diffkemp/diffkemp.py

```diff
@@ -668,7 +669,8 @@ def __init__(self, compare_output_dir, devel):
         # Path to the viewer dir containing data of the compared project.
         self.data_dir = None
         # Path to the viewer dir containing sources of the compared project.
-        self.source_dir = None
+        self.out_old_dir = None
+        self.out_new_dir = None
```

Comment:
Thanks, good point, for now I changed it to a prefix.

By a quick look at the codebase, I found out that we are mostly prefixing with `old`/`new`. The only inconsistency I discovered is with the naming of the snapshot directory (`snapshot_dir_old`/`snapshot_dir_new`) where we use suffixes instead. 

There is also inconsistency caused by using different terms for the same thing: `old`/`first`/`left`/`L` and `new`/`second`/`right`/`R` and prefixing/suffixing connected with it. Unifying this would require big refactoring as you mentioned.